### DISCUSSION TITLE
blog/ml-tasks

### DISCUSSION THREAD

**Discussion by giscus** [GawdOfGithub/dcc-blog#6] [General]
# blog/ml-tasks

https://blog.dccnita.in/blog/ml-tasks

**Comment by Deepraj87** [comment]
Do we need to create separate readme file for all the problem statement?

**Reply by GaneshRehan** [reply]
It is up to you. It is preferred to add a readme file and requirements file, and will also be scored by it. It can act as your extra credit for your work. And yes, it has to be added separately for each problem statement if done so.
